test/blackbox-tests/test-cases/jsoo/no-check-prim.t/run.t

```diff
@@ -56,7 +56,6 @@ Compilation using jsoo
   >   (default (disable_dynamically_linked_foreign_archives true)))
   > EOF
   $ dune build --display short bin/technologic.bc.js @install --profile dev
-   js_of_ocaml bin/technologic.bc.runtime.js
```

Comment:
How come the runtime isn't built here anymore?

Reply:
I don't know why it was rebuild here. It was already build on line 5